test_chatgpt-shell.el

```diff
@@ -1,22 +1,5 @@
 (require 'ert)
 
-;;; chatgpt-shell-model-version
```

Comment:
`chatgpt-shell-model-versions`, which is `chatgpt-shell-models` is grow up. I think this test case isn't fit it anymore?

Reply:
Yep. Let's remove.